File: Estrutura_Repetição/Java/Ex1072/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int dentro = 0, fora = 0;
        int quant = sc.nextInt();

        for (int i = 0; i < quant; i++) {
            int v = sc.nextInt();

            if (v >= 10 && v <= 20) {
                dentro += 1;
            } else {
                fora += 1;
            }
        }

        System.out.println(dentro + " in");
        System.out.println(fora + " out");

        sc.close();
    }
}

File: Estrutura_Repetição/C/Ex1072/Main.c
#include <stdio.h>

int main() {
    int dentro = 0, fora = 0, quant, v;
    scanf("%d", &quant);

    for (int i = 0; i < quant; i++) {
        scanf("%d", &v);

        if (v >= 10 && v <= 20) {
            dentro += 1;
        } else {
            fora += 1;
        }
    }

    printf("%d in\n%d out\n", dentro, fora);

    return 0;
}